### DISCUSSION TITLE
Updating Webpack With Extra Rules

### DISCUSSION THREAD

**Discussion by ErikCH** [vuepress/core#82] [Q&A] [answered]
I've been looking at the configuration for updating [Webpack](https://vuepress2.netlify.app/reference/bundler/webpack.html#configurewebpack) for Vuepress next. 

I'd like to configure these rules but I'm having problems adding them to the `configureWebpack` or `chainwebpack`?

```module: {
		rules: [
			{
				test: /\.m?jsx?$/,
				resolve: {
					fallback: {
						crypto: false,
					},
					fullySpecified: false,
				},
			},
]
```
How would I add this configuration in Vuepress for Webpack?

This is needed as a workaround for an issue we are having with [Amplify](https://github.com/aws-amplify/amplify-js/issues/7454)

**Comment by meteorlxy** [comment, marked as answer]
This might be what you need:

https://github.com/neutrinojs/webpack-chain#config-module-rules-uses-loaders-modifying-options

**Reply by ErikCH** [reply]
Thanks I"ll take a look at it
